beps/bep_0046.rst

```diff
@@ -42,7 +42,7 @@ Publishing content
 Publishers should issue a mutable ``put`` request when they want to notify
 consumers about an update of a torrent. The value of the payload ``v`` contains
 a dictionary with key ``ih`` (info hash) and value the 20 byte infohash of such
-torrent. Note that there is a 1-to-1 mapping between a mutable DHT item and a
```

Comment:
ah, right. technically, the "a" should be changed to an "an" as well. I can't help but think that there are much less ambiguous ways of saying this though. Short of just saying that many DHT items can point to the same torrent, would "relationship" be better than "mapping"?